src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -191,6 +191,15 @@ const FIELDS = {
           formatValue: value => (value ? value.toLocaleString('en-US') : '0'),
         },
       },
+      quantityRemaining: {
+        type: params => (params.subfield ? <LabelField {...params} /> : null),
+        label: 'Remaining',
+        fixedWidth: '75px',
+        fieldKey: '',
+        attributes: {
+          formatValue: fieldValue => (fieldValue.quantityRemaining ? fieldValue.quantityRemaining.toLocaleString('en-US') : fieldValue.quantityRemaining),
```

Comment:
formatValue has only one parameter which is value of current field, so this line should be: formatValue: value => (value ? (value.toLocaleString('en-US')) : value),